src/pages/instances/InstanceIps.tsx

```diff
@@ -13,11 +13,7 @@ const InstanceIps: FC<Props> = ({ instance, family }) => {
   return addresses.length ? (
     <ExpandableList
       items={addresses.map((item) => (
-        <div
-          key={item.address}
-          className="ip u-truncate"
-          title={`${item.address} (${item.iface})`}
-        >
+        <div key={item.address} className="ip u-truncate">
```

Comment:
I think we should keep the title, i.e. when truncated. See right-hand side in the screen below

![image](https://github.com/canonical/lxd-ui/assets/1155472/85976382-0e06-4102-890f-9cdb7420b583)